public/app/project-setup/test/config.stub.js

```diff
@@ -0,0 +1,4 @@
+// use empty object when running tests as the config
```

Comment:
no `window.yetu.config` when running tests with Karma, so as a global solution this file is included in the testing environment